jrcc-document-access-libs/src/main/java/ca/bc/gov/open/jrccaccess/libs/TransactionInfo.java

```diff
@@ -17,8 +18,9 @@
 	/**
 	 * the original sender (organization_
 	 */
+	@Value("${bcgov.access.sender}")
```

Comment:
are we sure that we need to get the sender from properties file for TransactionInfo?

Reply:
hmm this change isn't on the PR for me... I removed the `@Value()` annotation as we did not need it, so not sure why it is still showing up for you...
either way, yes I don't think we need that so will make sure it is not present!